Odata-docs/TOC.yml

```diff
@@ -79,6 +79,14 @@
                 href: /odata/webapi-8/fundamentals/ref-routing
           - name: Actions and Functions
             href: /odata/webapi-8/fundamentals/actions-functions
+          - name: Pagination
```

Comment:
Given @odero's comment elsewhere maybe swap "Pagination" with "Paging" on the TOC
Pagination => Paging
Pagination overview = Paging overview

Reply:
Should I also swap "Pagination => Paging" in the client docs TOC? Not sure if it might lead to dead links.

Reply:
I'd say only on the changes in this PR...

Reply:
@habbes Did you check if this change would result in dead links or not?

Reply:
The build output should report any broken links (for relative paths). I've also tested the paging links and haven't found a broken link so far.